README.rst

```diff
@@ -90,10 +91,13 @@ In e.g. your ``init.lua``:
 .. code-block:: lua
 
     require('lean').setup{
-        -- Enable abbreviation support?
-        --
-        -- false to disable, otherwise a table of options described below
+        -- Abbreviation support
         abbreviations = {,
+            -- Set one of the following to true to enable abbreviations
+            compe = false, -- nvim-compe source
+            snippets = false, -- snippets.nvim source
+            enable = false, -- built-in expander
```

Comment:
Can we call the option `builtin`, otherwise we've got `require'lean.abbreviations'.enable{enable = true}` as the lua API?